packages/peregrine/lib/talons/CategoryList/useCategoryList.js

```diff
@@ -0,0 +1,39 @@
+import { useEffect } from 'react';
+import { useQuery } from '../../hooks/useQuery';
+
+/**
+ * Returns props necessary to render a CategoryList component.
+ *
+ * @param {object} props
+ * @param {object} props.query - category data
+ * @param {string} props.id - category id
+ * @return {{ data: object, error: boolean, loading: boolean }}
```

Comment:
Is `error` a boolean or an `Error` object?

Reply:
I'll look and see how we're using it in `categoryList` and update the output accordingly.